cmake/support_catalyst.cmake

```diff
@@ -62,13 +79,28 @@ macro(FindCatalyst target_name)
                 ${HEADER_NAME}
                 URL                 https://raw.githubusercontent.com/PennyLaneAI/catalyst/${CATALYST_GIT_TAG}/runtime/include/${HEADER}
                 DOWNLOAD_NO_EXTRACT True
-                SOURCE_DIR          include
+                SOURCE_DIR          ../../include
+            )
+
+            FetchContent_MakeAvailable(${HEADER_NAME})
+        endforeach()
+
+        # Fetching tests hpp headers
+        set(TESTS_HEADERS TestUtils.hpp)
```

Comment:
Backend devices should not depend on any header files in `runtime/tests`. Could you remove this header from LQ?

Reply:
Done